app/mailers/user_mailer.rb

```diff
@@ -47,6 +47,11 @@ def user_promoted(user, role, url, settings)
     @image = logo_image
     @color = user_color
     @role = translated_role_name(role)
+    @admin_role = (role.get_permission("can_manage_users") ||
```

Comment:
No need for the conditional here since the get_permission method returns `true` or `false`. Instead: 
```
@admin_role = role.get_permission("can_manage_users") || 
              role.get_permission("can_manage_rooms_recordings") ||
              role.get_permission("can_edit_site_settings") ||
              role.get_permission("can_edit_roles")
```